index.bs

```diff
@@ -202,6 +202,14 @@ A [=WebTransport stream=] is one of <dfn for=stream>incoming unidirectional</dfn
    <td>Yes
    <td>Yes
   </tr>
+  <tr>
```

Comment:
I'm confused, hasn't it been already specified below as https://w3c.github.io/webtransport/#stream-signal-stop_sending?

Reply:
Thanks for catching that. I removed it again